packages/shortest/src/ai/client.ts

```diff
@@ -10,8 +10,9 @@ export class AIClient {
   private model: string;
   private maxMessages: number;
   private debugMode: boolean;
+  public basePrompt: string;
 
-  constructor(config: AIConfig, debugMode: boolean = false) {
+  constructor(config: AIConfig, debugMode: boolean = true) {
```

Comment:
I need to update the docs. We have a CLI arg called `pnpm shortest --debug-ai`. for your `debug.ts` solution please adjust it so that it will show your logs when that CLI arg is provided. Please show your logs along with the already implemented logs that are in place. You can refer to `cli/bin.ts`

Reply:
Yes, will include support for cli arguments to make debugging easier, also for enabling snapshots.